src/sdpLoops.jl

```diff
@@ -381,9 +382,10 @@ function sdp_hd_get_u(u_bounds::Array, x_bounds::Array,
         if constraints(t, x, u, w)&&is_next_state_feasible(next_state, x_dim,
                                                             x_bounds)
             admissible_u_w_count  += 1
-            ind_next_state = real_index_from_variable(next_state, x_bounds,
-                                                        x_steps)
-            next_V_x_w = cost(t, x, u, w) + Vitp[ind_next_state...]
+            #ind_next_state = real_index_from_variable(next_state, x_bounds,x_steps)
```

Comment:
dead comment?

Reply:
real_index_from_variable return real number, so cannot to use of index.
index_from_variable return integer index

Reply:
ok